src/content/protocols/aave.md

```diff
@@ -0,0 +1,916 @@
+
+### Aave Governance
+
+_Aave Governance_ refers to Aave v3's onchain governance system which controls contract upgrades as well as other critical permissions as outlined above. We discuss the governance process itself in the _Exit Window_ [section](#exit-window) and here focus on upgradeability and control in this subsystem itself. This governance process is implemented in the `GovernanceV3` contract which is fully upgradeable through a permissionless governance proposal. Hence, an unintended proposal could change the _Aave Governance_ system and reassign its control to a less robust or fully centralized setup. In order to mitigate this risk, upgrades to the `GovernanceV3` contract, as well as the `AAVE` token, require passing a 7 day _Exit Window_ (read more [here](#exit-window)). During this window, users can exit the Aave v3 protocol. Furthermore, the _Aave Governance v3 Guardian_ multisig account, adhering to the _Security Council_ requirements, can cancel the execution of unintended proposals. On the other hand, this control can be abused to censor regular proposals with majority support.
+
+Also note that the (current) implementation of `GovernanceV3` enables a native multi-chain governance process. Specifically, this process enables proposals to designate a different chain for holding governance votes. While Aave v3's proprietary cross-chain messaging protocol, called `a.DI`, itself does not exhibit centralized control, it makes the governance process susceptible to control over the designated chain itself.
+
+> Upgradeability score: High
+
+## Autonomy
+
+### Oracle and Prices
+
+The Aave V3 protocol relies on Chainlink oracle feeds to price collateral and borrowed assets in the system.
+
+The protocol does currently have limited validation on asset prices provided by Chainlink. This checks include upper caps for stablecoins and LSTs and a sanity check that the price is above 0 for all assets. If the reported price by the price feed was below 0, a fallback oracle would be queried.
```

Comment:
Typo in "This checks include upper caps for stablecoins and LSTs ", should be "These checks include upper caps for stablecoins and LSTs "